### ISSUE TITLE
Unable to start controller on rock64 board

### ISSUE THREAD

**Issue by mjnovice** [apache/openwhisk-deploy-kube#542]
Am facing https://github.com/apache/openwhisk-deploy-kube/issues/272
These are my kafka logs, This https://pastebin.com/MRChZYCv are my kafka logs. Let me know if more information is needed.

**Comment by dgrove-oss** [comment]
recording controller log from slack DM:
```
picocluster@pc0:~$ kubectl logs --follow  owdev-controller-0 -n openwhisk
[2019-10-26T17:50:15.819Z] [INFO] Slf4jLogger started
[2019-10-26T17:50:17.830Z] [INFO] Started the Kamon StatsD reporter
[2019-10-26T17:50:17.841Z] [INFO] Loaded metric reporter [kamon.statsd.StatsDReporter]
[2019-10-26T17:50:19.671Z] [INFO] [#tid_sid_unknown] [Config] environment set value for limits.triggers.fires.perMinute
[2019-10-26T17:50:19.677Z] [INFO] [#tid_sid_unknown] [Config] environment set value for limits.actions.sequence.maxLength
[2019-10-26T17:50:19.686Z] [INFO] [#tid_sid_unknown] [Config] environment set value for limits.actions.invokes.concurrent
[2019-10-26T17:50:19.689Z] [INFO] [#tid_sid_unknown] [Config] environment set value for limits.actions.invokes.perMinute
[2019-10-26T17:50:19.693Z] [INFO] [#tid_sid_unknown] [Config] environment set value for runtimes.manifest
[2019-10-26T17:50:19.697Z] [INFO] [#tid_sid_unknown] [Config] environment set value for kafka.hosts
[2019-10-26T17:50:19.701Z] [INFO] [#tid_sid_unknown] [Config] environment set value for port
[2019-10-26T17:50:22.919Z] [INFO] [#tid_sid_unknown] [KafkaMessagingProvider] topic completed0 already existed
[2019-10-26T17:50:23.965Z] [INFO] [#tid_sid_unknown] [KafkaMessagingProvider] topic health already existed
[2019-10-26T17:50:24.965Z] [INFO] [#tid_sid_unknown] [KafkaMessagingProvider] topic cacheInvalidation already existed
[2019-10-26T17:50:25.858Z] [INFO] [#tid_sid_unknown] [KafkaMessagingProvider] topic events already existed
[2019-10-26T17:50:27.401Z] [INFO] [#tid_sid_controller] [Controller] starting controller instance 0 [marker:controller_startup0_counter:7748]
[2019-10-26T17:50:32.531Z] [INFO] [#tid_sid_dispatcher] [MessageFeed] handler capacity = 128, pipeline fill at = 128, pipeline depth = 256
[2019-10-26T17:50:33.530Z] [INFO] [#tid_sid_unknown] [Controller] Shutting down Kamon with coordinated shutdown
[2019-10-26T17:50:33.735Z] [INFO] Message [org.apache.openwhisk.core.connector.MessageFeed$FillCompleted] from Actor[akka://controller-actor-system/user/$b#1260801435] to Actor[akka://controller-actor-system/user/$b#1260801435] was not delivered. [1] dead letters encountered. If this is not an expected behavior, then [Actor[akka://controller-actor-system/user/$b#1260801435]] may have terminated unexpectedly, This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[2019-10-26T17:50:33.812Z] [INFO] [#tid_sid_loadbalancer] [ShardingContainerPoolBalancerState] managedFraction = 0.9, blackboxFraction = 0.1
Exception in thread "main" java.lang.IllegalStateException: cannot create children while terminating or terminated
    at akka.actor.dungeon.Children.makeChild(Children.scala:285)
    at akka.actor.dungeon.Children.attachChild(Children.scala:47)
    at akka.actor.dungeon.Children.attachChild$(Children.scala:46)
    at akka.actor.ActorCell.attachChild(ActorCell.scala:445)
    at akka.actor.ActorSystemImpl.actorOf(ActorSystem.scala:783)
    at org.apache.openwhisk.core.loadBalancer.ShardingContainerPoolBalancer.<init>(ShardingContainerPoolBalancer.scala:216)
    at org.apache.openwhisk.core.loadBalancer.ShardingContainerPoolBalancer$.instance(ShardingContainerPoolBalancer.scala:359)
    at org.apache.openwhisk.core.controller.Controller.<init>(Controller.scala:114)
    at org.apache.openwhisk.core.controller.Controller$.start(Controller.scala:265)
    at org.apache.openwhisk.core.controller.Controller$.main(Controller.scala:212)
    at org.apache.openwhisk.core.controller.Controller.main(Controller.scala)
```

**Comment by dgrove-oss** [comment]
The 5 second gap here: 
```
[2019-10-26T17:50:27.401Z] [INFO] [#tid_sid_controller] [Controller] starting controller instance 0 [marker:controller_startup0_counter:7748]
[2019-10-26T17:50:32.531Z] [INFO] [#tid_sid_dispatcher] [MessageFeed] handler capacity = 128, pipeline fill at = 128, pipeline depth = 256
[2019-10-26T17:50:33.530Z] [INFO] [#tid_sid_unknown] [Controller] Shutting down Kamon with coordinated shutdown
```
Suggests that the controller is failing to respond to a liveness probe (which defaults to 5 seconds).  You can confirm this by adding the stanza below to you mycluster.yaml and redeploying. 
```
probes:
  controller:
    livenessProbe:
      initialDelaySeconds: 300
```
A 300 second delay should be enough to either (a) let the system come up because something is slower than expected on your rock64 cluster or (b) allow enough time to poke around inside the container to maybe see why the liveness probe is failing.

**Comment by mjnovice** [comment]
Increasing the liveness probe fixed the problem. Thanks @dgrove-oss and @rabbah

**Comment by dgrove-oss** [comment]
a PR would be great; thanks!

**Comment by mjnovice** [comment]
https://github.com/apache/openwhisk-deploy-kube/pull/543

**Comment by dgrove-oss** [comment]
merged #543
